packages/xstate-analytics/test/analytics.test.ts

```diff
@@ -81,29 +81,18 @@ describe('@xstate/analytics', () => {
         "count": 3,
         "transitions": Object {
           "": Object {
-            "{\\"type\\":\\"xstate.init\\"}": Object {
-              "count": 1,
-              "currentWeight": 1,
-              "relativeWeight": 1,
-              "state": "{\\"value\\":\\"green\\",\\"context\\":{}}",
-              "weight": 0.3333333333333333,
-            },
-          },
-          "{\\"value\\":\\"green\\",\\"context\\":{}}": Object {
             "{\\"type\\":\\"TIMER\\"}": Object {
-              "count": 1,
-              "currentWeight": 1,
-              "relativeWeight": 1,
+              "count": 2,
```

Comment:
Hm, it's somewhat surprising that this has been affected anyhow by this PR. Do you know why that is?

Reply:
👋

Reply:
Disregard this, I want to rework `@xstate/analytics` (very old project) before releasing it.

Reply:
The reason this has been affected is because it was using `state.history`, but now we have to use something different (which I added).